gpu/containers/include/pcl/gpu/containers/device_array.h

```diff
@@ -102,31 +102,33 @@ namespace pcl
             void upload(const T *host_ptr, std::size_t size);
 
             /** \brief Uploads data from CPU memory to internal buffer.
-             * Returns false if device_offset < device_begin_offset
+             * Returns false if device_begin_offset + num_elements > size of array
+             * Please noote that in contrast to the other upload function, this function
```

Comment:
```suggestion
             * \note In contrast to the other upload function, this function
```

Reply:
Thanks for the comment - I will modify the code accordingly!

Reply:
This should be fixed - thanks for the comment!